src/module.cpp

```diff
@@ -34,17 +35,23 @@ std::string Module::Compile() {
 		output += name + "_" + c.first + " := " + std::to_string(c.second) + ";\n";
 	}
 	for (const auto &reaction : reactions) {
-		for (const auto &specie : reaction.reactants) {
-			if (specie.second != 1) {
-				output +=
-						std::to_string(specie.second) + name + "_" + specie.first + " + ";
-			} else {
-				output += name + "_" + specie.first + " + ";
+		if (!reaction.reactants.empty()) {
+			for (const auto &specie : reaction.reactants) {
+				if (specie.first.empty()) {
+					output += "0";
+				} else if (specie.second != 1) {
+					output +=
+							std::to_string(specie.second) + name + "_" + specie.first + " + ";
+				} else {
+					output += name + "_" + specie.first + " + ";
```

Comment:
We should probably move the mapping of species name somewhere else at some point